pages/common/jc.md

```diff
@@ -0,0 +1,20 @@
+# jc
+
+> A utility to convert the output of many commands to JSON.
+> More information: <https://github.com/kellyjonbrazil/jc>.
+
+- Convert command output to JSON via pipe:
+
+`ifconfig | jc --ifconfig`
```

Comment:
I think `ifconfig` here can be changed right. If so, I think we should use bracket to indicate that this part can be change.

```suggestion
`{{ifconfig}} | jc {{--ifconfig}}`
```

Reply:
Ah yes, that works!